### ISSUE TITLE
Overscan Limitations Woes

### ISSUE THREAD

**Issue by moriel5** [farmerbb/SecondScreen#78]
I'm not sure what, if anything can be done about this, however I'll put it up regardless, at the very least for future reference.

I have an old 1080p Pilot TV from around a decade ago, which is on its way to the grave (though that could be completely unrelated to this), and had a pretty horrible overscan (with SecondScreen values needing to be set to 40, for both top and bottom (that are somehow controlled by the Right and Left variables) with no option to fix (perhaps with a technician remote, however I could not find any solution to this model, which appears to have never received proper support), though that is only useful for portrait mode), and I just got myself a USB-C to HDMI converter (relying on USB alternate mode (requiring at least DisplayPort 1.1a) for testing with my Razer Phone 2 (still on stock until I find time to flash a custom ROM).

Unfortunately, there appears to be only so much SecondScreen can do with Android's limitations (perhaps with root things will be better), since the overscan settings only help with the current orientation (perhaps separate settings for portrait and landscape could help, though that would bring in it's own challenges, since it would essentially mean changing the display preferences each time the orientation would change, which could be annoying to some), and media apps such as VLC, which stream the content to the display, would have the profile completely ignored while streaming (casting) the content (at least with VLC there is a manual override to just clone the display, which respects the set profile).

In short, the current limitations I have personally seen:
- Overscan compensation is limited to the current settings, which are only truly ideal for one orientation at a time.
-  Profile settings are ignored when casting.
